emulator/app/src/main.rs

```diff
@@ -358,21 +360,30 @@ fn run(cli: Emulator, capture_uart_output: bool) -> io::Result<Vec<u8>> {
     let flash_ctrl_event_irq = pic.register_irq(CaliptraRootBus::FLASH_CTRL_EVENT_IRQ);
     let flash_controller = DummyFlashCtrl::new(
         &clock.clone(),
-        None,
+        Some(PathBuf::from("primary_flash")), // TODO: make this configurable
         flash_ctrl_error_irq,
         flash_ctrl_event_irq,
     )
     .unwrap();
 
-    let auto_root_bus = AutoRootBus::new(
+    let auto_root_bus = Rc::new(RefCell::new(AutoRootBus::new(
```

Comment:
I thought you were going to remove this `Rc<RefCell<>>`? (It adds some overhead, so we would want to remove it if we can.)

Reply:
sorry for missing this. Just updated.